src/SonarQube.Client/Models/Connections.cs

```diff
@@ -90,24 +85,22 @@ object ICloneable.Clone()
 
         #region IDisposable Support
 
-        protected virtual void Dispose(bool disposing)
+        public void Dispose()
         {
             if (!this.isDisposed)
             {
-                if (disposing)
-                {
-                    this.Password?.Dispose();
-                }
-
+                this.Password?.Dispose();
                 this.isDisposed = true;
             }
         }
-
-        public void Dispose()
-        {
-            // Do not change this code. Put cleanup code in Dispose(bool disposing) above.
-            this.Dispose(true);
-        }
         #endregion
     }
+
+    public class ConnectionDTO
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename class 'ConnectionDTO' to match camel case naming rules, consider using 'ConnectionDto'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS101)